python/lsst/cp/pipe/cpDarkTask.py

```diff
@@ -37,14 +37,14 @@ class CpDarkConnections(pipeBase.PipelineTaskConnections,
     inputExp = cT.Input(
         name="cpDarkISR",
         doc="Input pre-processed exposures to combine.",
-        storageClass="ExposureF",
```

Comment:
Whys is this change necessary? (and below; from "ExposureF" to "Exposure". )

Reply:
You should only use ExposureF if you absolutely require 32-bit float exposures. If the next task doesn't care what type of Exposure it gets then it's better to be generic.